ArcGISRuntimeSDKQt_CppSamples/EditData/DeleteFeaturesFeatureService/DeleteFeaturesFeatureService.cpp

```diff
@@ -134,8 +134,9 @@ void DeleteFeaturesFeatureService::connectSignals()
     {
       Feature* feat = iter.next();
       // emit signal for QML
-      QString featureType = feat->attributes()->attributeValue("typdamage").toString();
-      m_mapView->calloutData()->setTitle("<br><b><font size=\"+2\">" + featureType + "</font></b>");
+      const QString featureType = feat->attributes()->attributeValue("typdamage").toString();
+      //! Html tags used to bold and increase pt size of callout title.
```

Comment:
for a comment you don't need the "!" (these are for snippets)

Reply:
Didn't intend that - force of habit!